<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewContentWindow</class>
 <widget class="QMainWindow" name="LogViewContentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>70</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Log View Contents</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Currently Selected View</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameTextBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="typeLabel">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="typeTextBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="logViewsBtn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Log Contents</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>logViewsBtn</zorder>
      <zorder>horizontalSpacer</zorder>
      <zorder>logViewsBtn</zorder>
      <zorder>typeLabel</zorder>
      <zorder>typeTextBox</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>nameTextBox</zorder>
      <zorder>nameLabel</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
